w04-associations-and-auth/d3-homework/README.md

```diff
@@ -0,0 +1,122 @@
+# Project 1 Planning
+
+## [Requirements](https://github.com/sf-wdi-22-23/modules/tree/master/w04-associations-and-auth/project-1)
```

Comment:
This link isn't valid yet, but I think it will be once @bgveenstra 's PR is merged.  We should double-check though.